src/styles/blocks/header.scss

```diff
@@ -0,0 +1,37 @@
+.header {
+  box-sizing: border-box;
+  height: 100vh;
+  background-image: url(../images/header-bg-mobile.png);
```

Comment:
the quality of your image is poor.
This is because you are using the mobile desktop version
Change image for the header and footer.

<img width="1453" alt="Знімок екрана 2024-11-04 о 15 02 06" src="https://github.com/user-attachments/assets/0e62b415-d1ac-473b-b31d-e715e2aa2b82">